src/main/java/seedu/task/model/util/DateParser.java

```diff
@@ -19,26 +19,26 @@
 
     //@@author A0164103W
     private static int parsedDate[] = new int[3];
-    private static final int DAY_INDEX = 0;
-    private static final int MONTH_INDEX = 1;
-    private static final int YEAR_INDEX = 2;
-    private static final int SUN_INDEX = 1;
-    private static final int MON_INDEX = 2;
-    private static final int TUE_INDEX = 3;
-    private static final int WED_INDEX = 4;
-    private static final int THU_INDEX = 5;
-    private static final int FRI_INDEX = 6;
-    private static final int SAT_INDEX = 7;
+    private static final int INDEX_DAY = 0;
+    private static final int INDEX_MONTH = 1;
+    private static final int INDEX_YEAR = 2;
```

Comment:
I know these were here before, but I think this `parsedDate` array should really be an object. You could make a small class with just a day, month, and year field rather than use this array with arbitrary indexes.